### ISSUE TITLE
See if we can get this running under PY3 via WxPython Phoenix

### ISSUE THREAD

**Issue by facelessuser** [facelessuser/Rummage#78]
Not holding my breath, but if this is mostly working, except for the usual PY2 vs PY3 compatibility issues, then it would be nice to have the code base ready for PY3 support as ultimately, that is the direction I'd like to go.

**Comment by facelessuser** [comment]
This does not look reliable or easy.  I'm going to wait on this.
